apps/src/localization/entrypoint.js

```diff
@@ -0,0 +1,225 @@
+/**
+ * The loading point for Localize and our localization engine.
+ *
+ * This should be the first code run in our application <head> and will be
+ * responsible for loading and initializing LocalizeJS.
+ */
+
+import {get, set} from 'js-cookie';
+
+/**
+ * The current course listing and a mapping between them and Localize project
+ * keys. This is a temporary measure for now.
+ */
+const csd_prefixes = [
+  '/courses/csd-2024',
+  '/s/csd1-2024',
+  '/s/csd2-2024',
+  '/s/csd3-2024',
+  '/s/csd4-2024',
+  '/s/csd5-2024',
+  '/s/csd6a-2024',
+  '/s/csd6b-2024',
+  '/s/csd7-2024',
+  '/s/csd-post-survey-2024',
+];
+
+const csf_prefixes = [
+  '/s/k5-unplugged',
+  '/s/express-2024',
+  '/s/pre-express-2024',
+];
+
+const donor_prefixes = [
+  '/s/k5-ai-data-2024',
+  '/s/elementaryai-2024',
+  '/s/getting-started-with-code',
+  '/s/k5howaimakesdecisions',
+  '/s/3-5gamedesign-2024',
+  '/courses/elementaryai-2024',
+  '/s/elem-game-design-2024',
+  '/courses/3-5gamedesign-2024',
+  '/s/getting-started-with-code',
+  '/s/k5howaimakesdecisions',
+  '/s/getting-started-with-code-game-design',
+  '/s/k5plgamedesign',
+];
+
+const aif_prefixes = [
+  '/courses/artificial-intelligence-foundations-2025',
+  '/s/aif1-2025',
+  '/s/aif2-2025',
+  '/s/aif3-2025',
+  '/s/aif4-2025',
+  '/s/aif5-2025',
+  '/s/aif6-2025',
+];
+
+const prefixes = {
+  MlKri360o3v2T: csd_prefixes,
+  '3vPUSGZrdllW2': csf_prefixes,
+  I0P5RaUEW8s5h: donor_prefixes,
+  zM53S8yC4TNgU: aif_prefixes,
+};
+
+const experiments = JSON.parse(window.localStorage.experimentsList || '[]');
```

Comment:
`localStorage` might throw a security error or be `null`, consider using `safeGetLocalStorage` in utils.js